index.html

```diff
@@ -0,0 +1,264 @@
+            </li>
+            <li>
+              <img src="images/Apple-logo-transparent.png" alt="" class="logo" />
+            </li>
+          </ul>
+          <ul class="ul_partners">
+            <li>
+              <img src="images/White-google-logo.png" alt="" class="logo" />
+            </li>
+            <li><img src="images/logo-Grown-Logo.png" alt="" class="logo" /></li>
+          </ul>
+        </div>
+      </div>
+      <div class="footer" id="footerHome">
+        <img src="images/v-icon-3.jpg" alt="" class="img_logo2">
+        <h2 class="title_footer">
+          The Vitron community
+        </h2>
+        <p class="p_footer">2010-2022 Vitron company, somme right reserved</p>
+      </div>
+    </section>
+      
+    <!-- </section> -->
+
+    <section class="About_section" id="About_section">
+      <div class="about_Intro">
+        <h2 class="Intro">"Hello, make it bounce with Vitron"</h2>
+      <h2 class="title_about">THE VITRON COMMUNITY SAYS HELLO.</h2>
+      <p class="p_description" id="p_desc2">
+        Vitron is an American company that manufactures audio hardware, including
+        loudspeakers and headphones and some Softwares. Vitron serves the consumer home and
+        professional market. The professional market includes studios,
+        installed/tour/portable sound, cars, music production, DJ, cinema
+        markets, etc. Vitron is owned by Harman International. JBL was founded by
+        James Vesta Lansing (1952–2019), an American audio engineer and
+        loudspeaker designer best known for establishing two audio companies
+        that bear his name, Altec Z and Vitron, the latter taken from his
+        initials.
+      </p>
+      <p class="contact_us">
+        Please contact us per Email for any furter questions About Vitron!
+        <br><span class="Email_us">vitrontechonologie@vtr.com</span>
+      </p>
+      </div>
+      <h2 class="titleAbout2">The Vitron Technologie</h2>
+      <div class="line_red"></div>
+      <p class="desc_logo">
+        The vitron logo was decided through the logo competition during summer
+        2019.
+      </p>
+      <div id="logo_big">
+        
+        <img src="images/v-icon-3.jpg" alt="" class="img_logo">
+        <h2 class="h2_logo">Vitron Tech</h2>
+      </div>
+      <div class="line_grey100"></div>
+
+      <h3 class="see_past_title">See our last 2 devices</h3>
+      <div class="line_red"></div>
+      <p class="p_see_past">Take a look at our last 2 devices </p>
+      <div class="see_past_wrapper">
+        <div class="past_card" id="past_card1">
+          <div class="overlay_red"></div>
+          <h2 class="year_last">2010</h2>
+          <p class="p_past_card">The votronXPZ</p>
+        </div>
+        <div class="past_card" id="past_card2">
+          <div class="overlay_red"></div>
+          <h2 class="year_last">2015</h2>
+          <p class="p_past_card">The votronXPZplus</p>
+        </div>
```

Comment:
- [ ] Kindly make these paragraphs displayed properly so all words appear and decrease the spacing between year and paragraph to match the Behance design.

| Your design | Behance design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/163589841-050ee892-e6d0-44cd-87e9-f72a0354ffae.png) | ![image](https://user-images.githubusercontent.com/24703171/163590156-2ea15d2e-6475-4b73-9206-7145ae585aa9.png) |